src/components/contextual/pages/pools/types/index.ts

```diff
@@ -0,0 +1,16 @@
+export type UserGuageShare = {
```

Comment:
Not sure if there's much point nesting this file here? Just have a types.ts file under pools.